main.js

```diff
@@ -0,0 +1,109 @@
+import {
+    app,
+    BrowserWindow,
+    // Menu
+} from 'electron';  // eslint-disable-line
+// const path = require('path');
+// const url = require('url');
+// const electron = require('electron')
+
+let win = null;
+
+
+function createWindow() {
+    // Initialize the window to our specified dimensions
+    win = new BrowserWindow({
+        width: 1000,
+        height: 600,
+    });
+    // Specify entry point
+    win.loadURL('http://localhost:3000');
+
+
+    // build menu from template
+    /*  const mainMenu = Menu.buildFromTemplate(mainMenuTemplate);
+      Menu.setApplicationMenu(mainMenu); */
+
+    // Show dev tools
+    win.webContents.openDevTools();
```

Comment:
This is useful when debugging, however, our client won't be debugging and therefore doesn't need this window open. What would actually be really nice is if you could make a variable at the top or something called DEBUG, its default will be false but while we're developing, we can switch it to true to better help us debug.